### DISCUSSION TITLE
调查和投票：最想听的课

### DISCUSSION THREAD

**Discussion by DViridescent** [coldercode/course#2] [投票]
你可以通过多种方式（issue、discussion、comment、线下沟通）提出你的需求，经过我们的评估后会将候选课程加入投票。

票数较高的课程可能会优先开始。

https://github.com/ColderCode/Course/discussions/6 本页面移动到此处

**Comment by VincentXGao** [comment]
- [x] Visual Studio 的debug姿势

**Reply by DViridescent** [reply]
有！第四课讲！

**Reply by Gzxyouzi** [reply]
怎么写log，怎么计算代码运行时间~

**Comment by Xiao-LJ** [comment]
- [ ] 如何**恰当地**使用ChatGPT/Copliot提高写代码姿势水平?

**Comment by Guanda0120** [comment]
我还要在投一票**面对对象**的程序设计~

**Comment by Guanda0120** [comment]
有没有可能将将数据库，比如有一些查找功能，我想比较多个项目的多个数据（需要将项目各个指标组成的特征向量归一化并用夹角推荐算法计算最近的向量）。应该clone到本地还是在mysql完成操作呢呀~

**Reply by silon-j** [reply]
数据库可以讲啊，但是大家对这个了解程度不同，可能要从头开始讲。你指的这个情况应该从数据库select 出来需要的数据，在后端代码里做二次运算，业务运算不应该放在mysql里进行

**Reply by Guanda0120** [reply]
这样子~学到了学到了~但是还是想了解一下数据库这块~期待开一个课讲一下~

**Comment by DViridescent** [comment]
这个页面似乎有问题，编辑后票数重置了。

而且这里不支持多选操作

**Comment by DViridescent** [comment]
https://github.com/ColderCode/Course/discussions/6 本页面移动到此处
